bndtools.core/src/bndtools/command/BndrunFilesParameterValues.java

```diff
@@ -21,44 +22,37 @@
 import org.eclipse.core.runtime.CoreException;
 import org.eclipse.core.runtime.IPath;
 
-import aQute.bnd.build.Workspace;
-import bndtools.central.Central;
+import aQute.lib.memoize.Memoize;
 
 public class BndrunFilesParameterValues implements IParameterValues {
 
 	static final String BNDRUN_FILE = "bnd.command.bndrunFile";
 
-	private static Map<String, String> bndrunFilesMap;
-	private static IResourceChangeListener	listener	= new InvalidateMapListener();
+	private IResourceChangeListener					listener	= new InvalidateMapListener();
+	private volatile Supplier<Map<String, String>>	bndrunFilesMap;
```

Comment:
By changing this to non-static, every command that shared this ParameterValues map is going to recalculate.  So right now we have 2 commands but in the future we may want more.  I don't think this is a huge performance hit right now, but if we have N commands, then there will be N copies of this class and N copies of the bndrunFilesMap.  Do y'all think that will be an issue?  Maybe not since there will not be an abundance of bndfiles usually.

Reply:
Got it! I can turn it back into static.

Reply:
Hey Ray, It seems the bndrunFilesMap didn't get switched back to static.  Change of plans?

Reply:
No sorry, will do momentarily!